skops/io/tests/test_external.py

```diff
@@ -303,9 +338,12 @@ def test_regressor(self, catboost, cb_regr_data, trusted, boosting_type):
 
         X, y = cb_regr_data
         estimator.fit(X, y, cat_features=[0, 1])
-        loaded = loads(dumps(estimator), trusted=trusted)
+        dumped = dumps(estimator)
+        loaded = loads(dumped, trusted=trusted)
         assert_method_outputs_equal(estimator, loaded, X)
 
+        visualize(dumped, trusted=trusted)
```

Comment:
these are going to print a bunch of stuff in the terminal when we run tests, shouldn't we be capturing the output? It also makes me realize, maybe we want to have an output arg for `visualize`?

Reply:
> these are going to print a bunch of stuff in the terminal when we run tests

pytest automatically captures stdout unless used with the `-s` option, no?

> maybe we want to have an output arg for `visualize`?

What would that argument do? We have the `sink` argument, in case someone wants to push the output to a logger, for instance.

Reply:
`sink` is not trivial to implement, it'd be nice if one could pass something like `os.devnull` and the output would be forwarded there.

I know by default pytest captures output, but even when we pass `-s` right now, there's almost no output, and this adds quite a bit there. If you think it's not worth fixing, I'm happy to merge as is.

Reply:
> `sink` is not trivial to implement, it'd be nice if one could pass something like `os.devnull` and the output would be forwarded there.

You should still be able to change `sys.stdout`, right?

> I know by default pytest captures output, but even when we pass -s right now, there's almost no output, and this adds quite a bit there.

I see. For `test_external.py`, I made a change to not print anything.

Reply:
CI is green, please review again